### ISSUE TITLE
Insert not reactive

### ISSUE THREAD

**Issue by vladblindu** [yogiben/meteor-autoform-modals#26]
The "insert" does not update the view as update and delete do.
Clients = new Meteor.Collection("client");

This is the schema:
var Clients_ss = new SimpleSchema({
  name: {
    type: String,
    label: "Client name",
    optional:false,
    max: 32
  },
  short: {
    type: String,
    label: "Code",
    optional:false,
    max:3,
    min:1
  }
});

Clients.attachSchema(Clients_ss);
var Clients_context = Clients_ss.namedContext("ClientsForm");

I have an edit/insert/delete button in the optional template provided by aldeed:tabular. Here's the template:

<template name="editablesClients">
    <div style="width: 100%; text-align: center;">
	{{#afModal class="btn-sm btn-primary" collection="Clients" operation="update" doc=_id title='<i class="glyphicon glyphicon-pencil text-primary"></i>&nbsp; Update record:'}}
		<i class="glyphicon glyphicon-pencil"></i>
	{{/afModal}}
	&nbsp;
	{{#afModal class="btn-sm btn-primary" collection="Clients" operation="remove" doc=_id title='<i class="glyphicon glyphicon-remove text-danger"></i>&nbsp; Delete record:'}}
		<i class="glyphicon glyphicon-remove"></i>
	{{/afModal}}
	&nbsp;
	{{#afModal class="btn-sm btn-primary" collection="Clients" operation="insert" title='<i class="glyphicon glyphicon-log-in text-primary"></i>&nbsp; Insert record:'}}
		<i class="glyphicon glyphicon-log-in"></i>
	{{/afModal}}
    </div>
</template>

Update and delete work perfect. What could be wrong with the insert statement?

Thanks a lot.

**Comment by yogiben** [comment]
Insert is definitely reactive. Are you checking the collection.fetch().find() in the browser console?

**Comment by vladblindu** [comment]
Hi there,

Till I received your mail I was sure that the:
```
 $('select').val($('select').val()).change();
```
line, I found in a post about the same issue was doing the trick in the onSuccess function. Getting your message, made me delete the line and... surprise,  it works.
I fixed a lot of things around my autoform implementation and especially vaIidation so, I think, that that was the source of the bug. 
Thanks a lot for your time.
